CHANGES.md

```diff
@@ -4,7 +4,13 @@ This release supports [version
 4](https://github.com/GaloisInc/mir-json/blob/master/SCHEMA_CHANGELOG.md#4) of
 `mir-json`'s schema.
 
-Nothing yet.
+## New Features
+
+* Add a `mir_exact_adt : String -> MIRAdt` command, which allows looking up MIR
+  ADTs by their full, instantiated names. `mir_exact_adt`'s use is discouraged
+  in favor of using `mir_find_adt` instead, but `mir_exact_adt` can be useful
+  in scenarios where `mir_find_adt` isn't expressive enough to look up
+  particular ADTs (e.g., ADTs that use const generics).
```

Comment:
I added this to the release notes for the release _after_ 1.4 under the assumption that we don't want to bother backporting this. If you feel differently, however, I'd be happy to move this to the 1.4 release notes instead in preparation for a backport.